Automattic/wp-calypso
# Tailored Flows: Remove Sensei Flow

Related to:
- https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

* Removed the Sensei tailored flow.
* Removed related components.

## Why are these changes being made?


* To clean up unused code.


## Testing Instructions



* Go to `/setup/sensei`
* Make sure it still redirects to `/plugins/sensei-pro`
* Check for console errors.
* Check for regressions.

## Pre-merge Checklist
